code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -527,14 +527,14 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	icon_state = "flechette"
 	hud_state = "shotgun_flechette"
 	bonus_projectiles_type = /datum/ammo/bullet/shotgun/flechette_spread
-	bonus_projectiles_amount = 4
+	bonus_projectiles_amount = 5
 	bonus_projectiles_scatter = 8
 	accuracy_var_low = 7
 	accuracy_var_high = 7
 	max_range = 15
 	damage = 20
 	damage_falloff = 0.5
-	penetration = 20
+	penetration = 75
```

Comment:
This was recently nerfed. We are moving away from AP because it produces spiky balancing and kills HvH. We are moving towards embedding flech.

Reply:
HvH should be largely irrelevant when discussing balance.

Reply:
It's not irrelevant, and it's not the only point.